### PR TITLE
Improve install docs

### PR DESCRIPTION
**Pull request by jvasile**
This is a series of patches that improve the install docs.

 * Mention newer versions of Debian.  Now we recommend current testing, Stretch.

 * Add an "apt install" line for postgres on Debian.

 * Specify user and pass via CLI rather than using an interactive script.

 * Tell user to check console output and fix errors.  Tell user to check for open ports.

 * Adjusted download url for jdbc

 * Use shell variables to make it easier to set version and password so the user doesn't have to visually parse a long heredoc in 4 different places.

 * Remove reference to build.properties.template because we don't use it.  Also, remove it from .gitignore

 * Clarify the gradlew vs gradle text.

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -312,7 +312,7 @@ EOF
 
    ```ShellSession
    $ cd ../{psm}/psm-app
-   $ ./gradlew cms-portal-services:build
+   $ ./gradle cms-portal-services:build
```
Comment (jasonaowen) [inline INSTALL.md]: We use `gradlew`, the gradle wrapper, as a way to ensure the version is the same across multiple machines. `gradlew` is checked in to the `psm-app` directory. See also [the Gradle Wrapper documentation](https://docs.gradle.org/current/userguide/gradle_wrapper.html).

File: INSTALL.md [hunk 2]
```diff
@@ -58,14 +58,15 @@ time, and will evolve as we understand it more.
...
-  a developer has successfully installed the PSM on Red Hat 7.3
-  Enterprise Linux. If that's not feasible for your environment, any
-  of the supported WildFly 10.1 operating systems should work, but our
-  ability to help troubleshoot issues that come up may be limited.
-  Once we test this on a few more platforms, we will expand the list
-  of compatible operating systems to include other Linux
-  distributions.
+- **Operating System**: we recommend Debian 8 (Jessie), which is the
+  old stable version.  We've also had success with Debian 9 (Stretch)
+  and Debian 10 (Buster).  A developer has also successfully installed
```
Comment (jasonaowen) [inline INSTALL.md]: Shall we simply recommend stretch? We have an issue open for that: #242. If you've tested it, that's good enough for me!

File: INSTALL.md (hunk 1 above)
Comment (jvasile) [inline INSTALL.md]: I saw that after I committed but by then I was on a subway.  I'll push a fix.

File: INSTALL.md (hunk 2 above)
Comment (jvasile) [inline INSTALL.md]: I was going to do that but wanted somebody else to agree and didn't want to hold up the PR on that convo.  So I agree and I will fix.

Comment (jvasile) [inline INSTALL.md]: Fixed.

File: INSTALL.md (hunk 1 above)
Comment (jvasile) [inline INSTALL.md]: Fixed and now with a comment so people don't think gradlew is a typo.

File: INSTALL.md [hunk 3]
```diff
@@ -58,14 +58,15 @@ time, and will evolve as we understand it more.
...
-- **Operating System**: we recommend the old stable Debian 8 (jessie), and
-  a developer has successfully installed the PSM on Red Hat 7.3
-  Enterprise Linux. If that's not feasible for your environment, any
-  of the supported WildFly 10.1 operating systems should work, but our
-  ability to help troubleshoot issues that come up may be limited.
-  Once we test this on a few more platforms, we will expand the list
-  of compatible operating systems to include other Linux
-  distributions.
+- **Operating System**: we recommend Debian 9 stable, also known as
+  Jessie.  If you prefer Debian testing, we have had success with
```
Comment (jasonaowen) [inline INSTALL.md]: Whoops, this should be stretch, not jessie. Also, I think Debian has a policy of keeping their release names lowercased, but all I could find when searching for that was a bunch of mailing list threads debating that policy...

**Pull request merged**